Take02/Views/Recipes/_AddEditIngredientPartial.cshtml

```diff
@@ -7,7 +7,7 @@
     <br />
     <input type="hidden" asp-for="@Model.Id" />
 
-    <input type="hidden" asp-for="@Model.Number" />
+    <input type="hidden" class="number" asp-for="@Model.Number" />
```

Comment:
This is necessary so that we can find the correct field in jquery without guessing